tests/issues.targets

```diff
@@ -544,6 +544,14 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- Unix arm64 or arm32 -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and ('$(BuildArch)' == 'arm64' or '$(BuildArch)' == 'arm') and '$(TargetsWindows)' != 'true' and '$(BuildTestsAgainstPackages)' != 'true'">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/CLR-x86-JIT/v2.1/b152292/b152292/*">
+            <Issue>20358</Issue>
```

Comment:
Do we really need another ItemGroup for arm32|arm64? 
I would just add the issue to Unix arm32 group below

Reply:
Well, I was trying to avoid duplication, but I can do that instead.